.gitignore

```diff
@@ -20,6 +20,10 @@ target/
 *.iml
 */.settings
 */.project
-docker/gaffer-wildfly/wars/*
+docker/gaffer-rest/wars/*
 .DS_Store
-**/.DS_Store
\ No newline at end of file
+**/.DS_Store
+**/Chart.lock
+linux-amd64/
+helm*.tar.gz
+**/*.swp*
```

Comment:
Not sure these lines should be added to .gitignore

We might want to consider deleting this file completely. I think it's left over from the previous incarnation of this repo, so it might be obsolete.

Reply:
Probably not worth deleting the file as there's still a fair amount of relevant IDE related stuff - and the Chart.lock line is still relevant. I don't think the linux-amd64 directory and helm*.tar.gz should be there though. They look like the result of external software installs. Could also move the charts/ entry out of the individual kubernetes .gitignore files and into this one.

Reply:
Just pushed a commit - how about that?

Reply:
The swp files are just vim files that get left over if your session crashes or you temporarily leave the session but do not close it.

Reply:
Yeah, I don't think they should be ignored though. If they exist then you should take action and either recover any edits that were in progress or choose to discard them, then delete the file.

Reply:
I'm fine with you adding those back in if you want to